pom.xml

```diff
@@ -143,6 +143,30 @@
   </profiles>
 
   <build>
+
+    <!-- src distribution -->
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>src</id>
+            <phase>package</phase>
```

Comment:
FYI: our build is getting pretty slow, and unnecessarily slowing developers down. The main cause is too much unnecessary plugins for that workflow. I think we should move all non-essential plugins to a profile.

That said, this is not isolated to this particular one. I'd keep as-is for now, but this is something we have to fix.

No action needed.